.buildkite/pipelines/pull_request/security_solution/playwright.yml

```diff
@@ -0,0 +1,32 @@
+steps:
+  - command: .buildkite/scripts/steps/functional/security_serverless_playwright.sh
+    label: 'Serverless Playwright - Security Solution Tests'
+    agents:
+      machineType: n2-standard-4
+      preemptible: true
+    depends_on:
+      - build
+      - quick_checks
+    timeout_in_minutes: 60
+    parallelism: 1
+    retry:
+      automatic:
+        - exit_status: '-1'
+          limit: 1
+    soft_fail: true
```

Comment:
This has caused a lot of confusion in the past - CI will appear to have a build failure.  Is this acceptable for the security team?

Operations will block on this globally, but we'll leave it up to you for now.

Reply:
> Operations will block on this globally

@jbudz what do you mean by block on this globally?

Reply:
If it's enabled in all pull requests.